terraform/trial_rg/services.tf

```diff
@@ -12,18 +22,29 @@ module "trial_app_service_site" {
   docker_image_tag    = var.site_image_tag
 
   settings = {
-    "WEBSITES_PORT"               = "8080" # The container is listening on 8080
+    "SPRING_APPLICATION_NAME"                 = "site-service"
+    "SPRING_PROFILES_ACTIVE"                  = var.spring_profile
+    "SPRING_CLOUD_CONFIG_LABEL"               = var.spring_config_label
+    "SERVER_PORT"                             = "80"
+    "WEBSITES_PORT"                           = "80"
+    "SPRING_CLOUD_CONFIG_DISCOVERY_ENABLED"   = "true"
+    "SPRING_CLOUD_CONFIG_DISCOVERY_SERVICEID" = "config-server"
```

Comment:
This is making the profiles redundant... should we do this?

Reply:
correct